server/routes/referrals/completionDeadlinePresenter.test.ts

```diff
@@ -4,15 +4,19 @@ describe('CompletionDeadlinePresenter', () => {
   describe('day, month, year', () => {
     describe('when the referral has no completion deadline and there is no user input data', () => {
       it('returns empty strings', () => {
-        const presenter = new CompletionDeadlinePresenter({
-          id: '1',
-          completionDeadline: null,
-          serviceCategory: {
+        const presenter = new CompletionDeadlinePresenter(
+          {
+            id: '1',
+            completionDeadline: null,
+            serviceCategoryId: 'b33c19d1-7414-4014-b543-e543e59c5b39',
+            complexityLevelId: null,
+          },
+          {
             id: 'b33c19d1-7414-4014-b543-e543e59c5b39',
             name: 'social inclusion',
-          },
-          complexityLevelId: null,
-        })
+            complexityLevels: [],
```

Comment:
Now that #45 is merged, could we create a factory for creating `ServiceCategory` instances, and update the tests to use that? (Easy way to find out all the places you need to change: add an extra dummy property to `ServiceCategory` and see all the places the compiler throws an error.)

Reply:
Done 😓

Reply:
Although as I rebased, it became difficult to commit smaller things in this PR (i.e. changes to the `ServiceCategoryFactory`, which I would have liked to be in their own commit), but I've spent so long merging and rebasing other branches over the past few days I don't really have the patience to sort this out...